bootstrap/services/storm-bundle.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "STORM",
+  "registerClass": "com.hortonworks.streamline.streams.cluster.register.impl.StormServiceRegisterer",
+  "serviceUISpecification": {
+    "fields": [
+      {
+        "uiName": "UI server hostname",
+        "fieldName": "uiServerHostname",
+        "isOptional": false,
+        "tooltip": "Hostname of Storm UI server",
+        "type": "string"
+      },
+      {
+        "uiName": "Nimbuses hostnames",
+        "fieldName": "nimbusesHostnames",
+        "isOptional": false,
+        "tooltip": "Hostnames of Nimbuses",
+        "type": "array.string"
+      },
+      {
+        "uiName": "storm.yaml",
+        "fieldName": "storm.yaml",
+        "isOptional": false,
```

Comment:
@HeartSaVioR Want to clarify your last comment. You said it will not be used for overriding storm config for topology submission. Should not we be using this for submitting topology, now that we have this?